docker-compose.base.yml

```diff
@@ -25,7 +25,7 @@ services:
         # Note: please keep the default version in sync across
         #       `posthog` and the `charts-clickhouse` repos
         #
-        image: ${CLICKHOUSE_SERVER_IMAGE:-clickhouse/clickhouse-server:23.6.1.1524}
+        image: ${CLICKHOUSE_SERVER_IMAGE:-clickhouse/clickhouse-server:23.11.2.11-alpine}
```

Comment:
Why `alpine`? it's about 15% smaller

Reply:
I'm assuming that since ClickHouse uses ICU for collation that this means the merge tree ordering should be stable if a volume is reused across these images?

Reply:
Clickhouse is really great about forwards compatibility of merge tree files on disk. You have to worry more about backwards compatibility, but even then it's really great and you can suppress things that will make it backwards incompatible (we've done that before) like with `compress_primary_key`. This is actually something I'll be looking to enable since we'd fail back to a version that supports it.

https://clickhouse.com/codebrowser/ClickHouse/src/Storages/MergeTree/MergeTreeIOSettings.h.html#54

Reply:
It's actually interesting - it won't rework the parts and so for some time you'll have a mix of parts written by different version of CH

Reply:
Long story short though, yeah ICU as collator

https://github.com/ClickHouse/ClickHouse/blob/66d2db52832a81aea43cda66a500d8b3369547ef/src/Common/config.h.in#L41
https://github.com/ClickHouse/ClickHouse/blob/66d2db52832a81aea43cda66a500d8b3369547ef/src/Columns/Collator.cpp#L34